src/main/kotlin/com/exactpro/th2/codec/ValidateException.kt

```diff
@@ -0,0 +1,25 @@
+package com.exactpro.th2.codec
+
+class ValidateException : CodecException {
+
+    var title: String? = null
+
+    constructor() : super()
+    constructor(title: String?) : super(title) {
+        this.title = title
+    }
+
+    constructor(cause: Throwable?) : super(cause)
+    constructor(title: String?, cause: Throwable?, enableSuppression: Boolean, writableStackTrace: Boolean) : super(
+        title,
+        cause,
+        enableSuppression,
+        writableStackTrace
+    ) {
+        this.title = title
+    }
+
+    constructor(title: String?, cause: Throwable?) : super(title, cause) {
+        this.title = title
+    }
```

Comment:
I think we can remove those constructors.
Also, I think this exception should be lightweight (it can be done by using Exception's constructor with `writableStackTrace` and `enableSuppression` set to `false`)